src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+public class Truck extends Machine{
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");

```

Comment:
Similarly, the message 'Truck stopped working' could be more informative by indicating what the Truck has finished doing or what state it is now in.